### PR TITLE
add w6 follow up themes

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: code_schemes/kakuma_adolescent_mothers_challenges.json [hunk 1]
```diff
@@ -3,6 +3,54 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "lack financial support",
+      "NumericValue": 1,
+      "StringValue": "lack_financial_support",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-45f338e8",
+      "CodeType": "Normal",
+      "DisplayText": "face rejection / discrimination",
```
Comment (IsaackMwenda) [inline code_schemes/kakuma_adolescent_mothers_challenges.json]:
```suggestion
      "DisplayText": "face rejection or discrimination",
```

File: code_schemes/dadaab_adolescent_mothers_challenges.json [hunk 2]
```diff
@@ -3,6 +3,46 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "pregnancy complications",
+      "NumericValue": 2,
+      "StringValue": "pregnancy_complications",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-786f0ac2",
+      "CodeType": "Normal",
+      "DisplayText": "discrimination, embarrassment & stigma",
```
Comment (IsaackMwenda) [inline code_schemes/dadaab_adolescent_mothers_challenges.json]:
```suggestion
      "DisplayText": "discrimination, embarrassment and stigma",
```

File: code_schemes/dadaab_adolescent_mothers_challenges.json [hunk 3]
```diff
@@ -3,6 +3,46 @@
   "Name": "DADAAB Adolescent Mothers Challenges",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-2eb181c4",
```
Comment (IsaackMwenda) [inline code_schemes/dadaab_adolescent_mothers_challenges.json]: nit: rearrange the codes as-is on the code scheme file.

**Review by IsaackMwenda — APPROVED** [review]

File: code_schemes/dadaab_adolescent_mothers_challenges.json (hunk 3 above)
Comment (DanielMwendwa) [reply code_schemes/dadaab_adolescent_mothers_challenges.json]: Updated, thanks.

**Pull request merged**